### ISSUE TITLE
Unable to send emails -- Connection timed out

### ISSUE THREAD

**Issue by pallatinus** [LukeSmithxyz/emailwiz#136]
I'm receiving emails just fine, but I'm having problems sending them.

I'm getting the following output with `journalctl -xe`:
```
postfix/smtp[11977]: connect to mail.emailprovider.com[185.70.40.102]:25: Connection timed out
postfix/smtp[11977]: 30244801A0: to=<sending@email.com>, relay=none, delay=29938, delays=29878/0.05/60/0, dsn=4.4.1, status=deferred (connect to mail.emailprovider.com[185.70.40.102]:25: Connection timed out)
```
Port 25 is open.
My TXT records seem to be correct.
According to intodns.com, there are no problems with my MX records.

**Comment by krieem** [comment]
I had the same issue. turned out Vultr block port 25 for new users and you need to contact them in order to unblock the port.

https://www.vultr.com/docs/what-ports-are-blocked

**Comment by pallatinus** [comment]
> I had the same issue. turned out Vultr block port 25 for new users and you need to contact them in order to unblock the port.
> 
> https://www.vultr.com/docs/what-ports-are-blocked

Apparently that's not the case. I've checked and the port is open.

**Comment by pallatinus** [comment]
On April 6, I received a Mail Delivery System reply:
[-- Attachment #1: Notification --]
                   │[-- Type: text/plain; charset=utf-8, Encoding: 8bit, Size: 0.5K --]
                   │
                   │This is the mail system at host mail.email.com.
                   │
                   │I'm sorry to have to inform you that your message could not
                   │be delivered to one or more recipients. It's attached below.
                   │
                   │For further assistance, please send mail to postmaster.
                   │
                   │If you do so, please include this problem report. You can
                   │delete your own text from the attached returned message.
                   │
                   │                   The mail system
                   │
                   │<name@emailprovider.com>: connect to mailsec.emailprovider.com[185.70.40.102]:25:
                   │    Connection timed out
                   │
                   │[-- Attachment #2: Delivery report --]
                   │[-- Type: message/delivery-status, Encoding: 7bit, Size: 0.4K --]
                   │
                   │Reporting-MTA: dns; mail.emailprovider.com
                   │X-Postfix-Queue-ID: 00433801AF
                   │X-Postfix-Sender: rfc822; sending@email.com
                   │Arrival-Date: Thu,  1 Apr 2021 16:06:00 +0000 (UTC)
                   │
                   │Final-Recipient: rfc822; name@emailprovider.com
                   │Original-Recipient: rfc822;name@emailprovider.com
                   │Action: failed
                   │Status: 4.4.1
                   │Diagnostic-Code: X-Postfix; connect to mailsec.emailprovider.com[185.70.40.102]:25:
                   │    Connection timed out

**Comment by pallatinus** [comment]
> I had the same issue. turned out Vultr block port 25 for new users and you need to contact them in order to unblock the port.
> 
> https://www.vultr.com/docs/what-ports-are-blocked

I've created a Firewall group on Vultr's website and enabled all the ports manually, and no success.

**Comment by pallatinus** [comment]
Surprisingly two out of the 6 emails that I send on April 1st, arrived on April 6. But in general I'm still unable to send emails.

**Comment by krieem** [comment]
> > I had the same issue. turned out Vultr block port 25 for new users and you need to contact them in order to unblock the port.
> > https://www.vultr.com/docs/what-ports-are-blocked
> 
> Apparently that's not the case. I've checked and the port is open.

You can't open the port by your own, even though it is open from your side. you have to open a support ticket to open the port and provide some information about your mail usage to ensure there will be no abuse of there service.
